meinberlin/apps/maptopicprio/api.py

```diff
@@ -0,0 +1,25 @@
+from django.db.models import Model
+from rest_framework.serializers import ModelSerializer
+
+from meinberlin.apps.maptopicprio.models import MapTopic
+from meinberlin.apps.maptopicprio.serializers import MapTopicSerializer
+from meinberlin.apps.topicprio.api import BaseTopicViewSet
+from meinberlin.apps.topicprio.api import ModuleMixin
+from meinberlin.apps.topicprio.api import PermissionInfoMixin
+from meinberlin.apps.topicprio.api import TopicFilterInfoMixin
+
+
+class BaseMapTopicViewSet(BaseTopicViewSet):
```

Comment:
this is not needed, just add the model and the serializer_class to `MapTopicViewSet`